docs/population.md

```diff
@@ -354,6 +364,19 @@ To make use of the urban type, the following data is needed:
 
 Then, you should be able to run the pipeline with the configuration explained above.
 
+### Filter household travel survey data
+
+The pipeine filter out by default observations, from household travel survey used, which live or work outside of the area used in the simulation.
```

Comment:
By default, the pipeline filters out observations from the HTS that correspond to persons living or working outside the configured area (given as departments or regions).